src/main/java/com/nextcloud/android/sso/api/AidlNetworkRequest.java

```diff
@@ -218,10 +247,128 @@ public void onThreadFinished(Thread thread) {
         return output;
     }
 
+    /**
+     * DO NOT CALL THIS METHOD DIRECTLY - use @link(performNetworkRequest) instead
+     *
+     * @param request
+     * @return
+     * @throws IOException
+     */
+    private ParcelFileDescriptor performNewAidlNetworkRequest(NextcloudRequest request,
```

Comment:
Maybe we should use some kind of versioning instead of `New`. E.g. `performAidlNetworkRequestV2`

Reply:
V2 is a good idea, I will change it everywhere.